hack/package-helm-charts.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -e
```

Comment:
Both of these seem redundant, can we just use the make target from the pipeline as well.

Reply:
makes sense

Reply:
done